eve/main.yml

```diff
@@ -285,6 +286,144 @@ stages:
           doStepIf: false
           alwaysRun: true
 
+  single-node-upgrade-test:
+    worker:
+      type: openstack
+      image: CentOS-7-x86_64-GenericCloud-1809.qcow2
+      flavor: m1.large
+      path: eve/workers/openstack-single-node
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
+          name: Report IP and install SSH keys
+          command: >
+            ip a &&
+            mkdir -p ~/.ssh &&
+            echo "%(secret:ssh_pub_keys)s" >> ~/.ssh/authorized_keys
+      - SetPropertyFromCommand: &set_prev_version
```

Comment:
I would prefer to define all this step in downgrade as downgrade will be the first step to run, so we can keep them here but declaration should be moved to downgrade step after

Reply:
yes, that makes absolute sense